dtls.c

```diff
@@ -337,6 +349,9 @@ dtls_init(void) {
 memarray_init(&dtlscontext_storage, dtlscontext_storage_data,
               sizeof(dtls_context_t), DTLS_CONTEXT_MAX);
 #endif /* RIOT_VERSION */
+#ifdef DTLS_ECC
+ uECC_set_rng(uecc_rng_function);
+#endif /* DTLS_ECC */
```

Comment:
Maybe that could be moved to "crypto.c". In my opinion, that would limit the scope of "ecc-related changes" to crypto.